### PR TITLE
Fix JSDoc annotations

### PR DESCRIPTION
**Pull request by hochhaus**

### CODE REVIEW

File: test/media/streaming_engine_integration.js [hunk 1]
```diff
@@ -522,6 +522,7 @@ describe('StreamingEngine', function() {
    * Initializes or switches to the given period.
    *
    * @param {shakaExtern.Period} period
+   * @return {{audio: shakaExtern.Stream, video: shakaExtern.Stream}}
```
Comment (joeyparrish) [inline test/media/streaming_engine_integration.js]: I think this should be {!Object.<string, shakaExtern.Stream>}

**Comment by hochhaus** [comment]
Thanks. Updated per your comment.

**Comment by hochhaus** [comment]
I actually updated again to `@return {!Object.<string, !shakaExtern.Stream>}` as that type is slightly tighter. Please let me know if you disagree.

File: test/media/streaming_engine_integration.js [hunk 2]
```diff
@@ -522,6 +522,7 @@ describe('StreamingEngine', function() {
    * Initializes or switches to the given period.
```
Comment (joeyparrish) [inline test/media/streaming_engine_integration.js]: This comment is out of date.  Can you please update to "Choose streams for the given period"?  Same in player_unit and streaming_engine_unit.

**Comment by hochhaus** [comment]
Updated the comments per your feedback.

PTAL

**Comment by joeyparrish** [comment]
Since shakaExtern.Stream is a typedef of a record type, it is already non-nullable by default, but there's nothing wrong with adding the exclamation point.

**Comment by hochhaus** [comment]
> it is already non-nullable by default

Thanks for the explanation. I didn't realize that.

**Comment by joeyparrish** [comment]
Yeah, sometimes Closure is more subtle than I would like.

**Pull request merged**